packages/compiler/test/parser.test.ts

```diff
@@ -730,11 +737,80 @@ describe("compiler: parser", () => {
   describe("invalid statement", () => {
     parseErrorEach([["@myDecN.)", [/Identifier expected/]]]);
   });
+
+  describe("doc comments", () => {
```

Comment:
I guess we don't really have that for other parsing but should we have some test that check the content not just make sure its parsing?

Reply:
Yes, we absolutely should, I was working on that and decided to put this up without finishing that before the sync up on refdoc so as not to be blocking anyone.

And we should fix the other parse tests too!

Reply:
I'll add the assertions on the parse result before merging this.

Reply:
Filed https://github.com/microsoft/cadl/issues/1314 for the rest.

Reply:
Added assertions. I still plan to add more cases in follow-up.